packages/server/test/complete.test.ts

```diff
@@ -450,17 +454,15 @@ describe('Fully qualified table names', () => {
     const expected = [expect.objectContaining({ label: 'table3' })]
     expect(result.candidates).toEqual(expect.arrayContaining(expected))
   })
-  test('complete table name by using fully qualified table name', () => {
+
+  test('not complete table name when not qualified', () => {
     const result = complete(
-      'SELECT * FROM tabl',
+      'SELECT * FROM TABL',
```

Comment:
question: Just noticed that this was lowercased. Any particular reason for that?

Reply:
Good point. I'll improve this test a bit